modules/notifications/notifications.php

```diff
@@ -602,6 +602,9 @@ function notification_status_change( $new_status, $old_status, $post ) {
 	
 	/**
 	 * Set up and set editorial comment notification email
+	 * 
+	 * @param WP_Comment $comment
+	 * @return boolean
```

Comment:
```suggestion
	 * @return boolean|null|void
```

Reply:
Could probably just make both of the early `returns` return the same thing to simplify this, either null for both or false for both. Not a big deal though.